astroquery/gaia/core.py

```diff
@@ -898,5 +899,25 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def rename_table(self, table_name=None, new_table_name=None, new_column_names_dict={},
+                     verbose=False):
+        """
+            This new method allows to update the column names of a user table.
+            header example: rename_table(table_name=old_table_name, new_table_name=new_table_name -optional-
+             , new_column_names_dict=[old_column1:new_column1, old_column2:new_colum2...])
```

Comment:
If this is meant to be an example for users to follow, it belongs in the
```
Example
-------
```
section below `Parameters`.